test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Csv.Tests.ps1

```diff
@@ -50,22 +50,47 @@ Describe "ConvertTo-Csv" -Tags "CI" {
     }
 
     It "Should return the type of data in the first element of the output array" {
-	$result = $testObject | ConvertTo-Csv
+	$result = $testObject | ConvertTo-Csv -IncludeTypeInformation
 
 	$result[0] | Should Be "#TYPE System.Management.Automation.PSCustomObject"
     }
 
     It "Should return the column info in the second element of the output array" {
```

Comment:
We could remove the test - it is dup test in line 28.
Or it is better move the checks and next test to the test in line 28 - there we can check a type, a header and data.

It is minor comment.

Reply:
@iSazonov Which test do you want to remove? the one you comment is on (lin 58) is not a duplicate of any of the tests. Line 28 is a delimiter test which needs to remain as its a different logic gate (parameter set).

the test on line 52 is now redundant with the the test on line 84, though.

Reply:
It seems we could try to refactor the tests but not in the PR.
So skip the minor comment.